checker/tests/src/tests/FenumTest.java

```diff
@@ -16,7 +17,7 @@ public FenumTest(File testFile) {
     }
 
     @Parameters
-    public static Collection<Object[]> data() {
-        return testFiles("fenum", "all-systems");
+    public static Collection<Object[]> getTestFiles() {
```

Comment:
Note, I think in the long run we should separated the tests and their corresponding input test files into two directories:
test 
testdata

I think it's confusing to have both the testdata and the tests that run against that data in the same folder at the same level.